src/main/webapp/app/exercises/programming/manage/programming-exercise.component.html

```diff
@@ -2,7 +2,9 @@
     <h4 class="d-flex mb-3 flex-wrap justify-content-end">
         <span class="mb-1 me-auto">
             <span *ngIf="course && !showHeading"> {{ course.title }} - </span>
-            <span *ngIf="programmingExercises && showHeading && !filter.isEmpty()">{{ getAmountOfExercisesString(filteredProgrammingExercises) }} / </span>
+            <span *ngIf="programmingExercises && showHeading && !filter.isEmpty() && programmingExercises.length"
+                >{{ getAmountOfExercisesString(filteredProgrammingExercises) }} /
```

Comment:
```suggestion
            <span *ngIf="programmingExercises && showHeading && !filter.isEmpty() && programmingExercises.length">
                {{ getAmountOfExercisesString(filteredProgrammingExercises) }} /
```
The same formatting change can then be applied here as well, probably.

Reply:
How did I miss that... (It's probably too early, thanks)